packages/prisma-client/prisma/schema.prisma

```diff
@@ -81,9 +81,10 @@ model Build {
   isDev  Boolean // exctly one is true per project
   isProd Boolean // at most one is true per project (none if not published)
 
-  breakpoints  String @default("[]")
-  styles       String @default("[]")
-  styleSources String @default("[]")
+  breakpoints           String @default("[]")
+  styles                String @default("[]")
+  styleSources          String @default("[]")
+  styleSourceSelections String @default("[]")
```

Comment:
Next PR should be removal of styleSelections from tree?

Reply:
> Next PR should be removal of styleSelections from tree?

plz add an issue about outdated fields removal

Reply:
We move all data from Tree table and remove it completely. So selective removal not necessary.